### ISSUE TITLE
Change preview icon in toolbar breaking functionality

### ISSUE THREAD

**Issue by arduanov** [sparksuite/simplemde-markdown-editor#100]
icons hardcoded in css

```
.editor-toolbar.disabled-for-preview a:not(.fa-eye):not(.fa-arrows-alt):not(.fa-columns) {
    pointer-events: none;
    background: #fff;
}
```

**Comment by WesCossick** [comment]
This is by design. CSS is the easiest way to handle disabling pointer events when a particular class is present on a parent. If you've manually changed the icon for the preview, you'll also need to add a quick CSS rule to handle it properly when other icons are disabled.

**Comment by arduanov** [comment]
I agree, but the documentation is not written about it. I spend time to solve this bug.

**Comment by WesCossick** [comment]
I look into ways of not hardcoding it to the class, which may change.

**Comment by WesCossick** [comment]
Fixed with commit a52a9a009803a75dab4b0760a89a6849669922b9
